controllers/fenceagentsremediation_controller.go

```diff
@@ -265,6 +277,16 @@ func (r *FenceAgentsRemediationReconciler) updateStatus(ctx context.Context, far
 	return nil
 }
 
+// checkAgentSupport return error if the agent name from the CR is not matching one of the available agents
+func (r *FenceAgentsRemediationReconciler) checkAgentSupport(agent string) error {
```

Comment:
I think this method should return a bool.
error is usually reserved for a use case when something unexpected went wrong.

Reply:
>error is usually reserved for a use case when something unexpected went wrong.

There are a few points in this function when an error might occur (file is not found, file is empty or we can't read the file). Do you suggest ignoring them or printing log error and exit error or (as it is right now) returning a unique error which will be checked and printed when `checkAgentSupport` is called

Reply:
I think you might refer to `checkFenceAgentsFileExists` method in main.
At the moment as far as I can see this method only returns an error in one place which will occur if the agent searched is not in the list.

Reply:
>I think you might refer to checkFenceAgentsFileExists method in main.

Correct, I got confused with the names.